src/Microsoft.Extensions.Diagnostics.HealthChecks/DefaultHealthCheckService.cs

```diff
@@ -75,10 +75,11 @@ internal class DefaultHealthCheckService : HealthCheckService
                             var result = await healthCheck.CheckHealthAsync(context, cancellationToken);
 
                             entry = new HealthReportEntry(
-                                result.Result ? HealthStatus.Healthy : registration.FailureStatus,
-                                result.Description,
-                                result.Exception,
-                                result.Data);
+                                status: result.Result ? HealthStatus.Healthy : registration.FailureStatus,
+                                description: result.Description,
+                                duration: stopwatch.GetElapsedTime(),
```

Comment:
I think it would be good to store `stopwatch.GetElapsedTime()` in a local here because it's used twice (L80 & L84). Normally I wouldn't care about this, but I want to avoid logging + health report entry having different time values.